build/docker/bats/Dockerfile

```diff
@@ -0,0 +1,66 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM alpine:3.8
+
+RUN apk add --no-cache \
+  bash \
+  curl \
+  git \
+  jq \
```

Comment:
`jq` does not appear to be used.

Reply:
Good catch, same with make.  Removing both.

Reply:
Removed jq and make in 7e56437...0f74e0d HEAD -> test_structure (forced update)